lib/2wp-utils.js

```diff
@@ -107,8 +107,11 @@ const createPegoutRequest = async (rskTxHelper, amountInRBTC, requestSize = 1) =
  */
 const isUtxoRegisteredInBridge = async (rskTxHelper, peginTxHash, expectedUxtosCount = 1) => {
     const bridgeState = await getBridgeState(rskTxHelper.getClient());
-    return bridgeState.activeFederationUtxos
-        .filter(utxo =>  utxo.btcTxHash === peginTxHash).length === expectedUxtosCount;
+    const foundUtxos = bridgeState.activeFederationUtxos.filter(utxo =>  utxo.btcTxHash === peginTxHash);
+    if(foundUtxos.length > 0 && foundUtxos.length !== expectedUxtosCount) {
+        throw new Error(`Found ${foundUtxos.length} utxo(s) with the pegin tx hash ${peginTxHash} in the bridge, but expected to find ${expectedUxtosCount} utxo(s).`);
+    }
+    return foundUtxos.length === expectedUxtosCount;
```

Comment:
guess this will always be true then?

Reply:
It will not. I replied to the comment above.